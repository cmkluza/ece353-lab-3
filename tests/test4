./sim-mips -b 1 1 1 ./tests/asm/test4.s ./tests/out/test4.out
./tests/sols/test4.sol