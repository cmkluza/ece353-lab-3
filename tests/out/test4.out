program name: ./tests/asm/test4.s
stage utilization: 0.619048  0.571429  0.857143  0.000000  0.714286 
register values 0  0  0  0  0  0  0  1  2  3  4  5  3  0  0  15  10  0  170  0  0  0  0  0  0  0  0  0  0  0  0  60
